### PR TITLE
fix(path-map): disallow nested tokens

### PR DESCRIPTION
**Pull request by barmac**
Related to https://github.com/bpmn-io/bpmn-js/pull/1497

### CODE REVIEW

File: packages/dmn-js-drd/src/draw/PathMap.js [hunk 1]
```diff
@@ -173,8 +173,8 @@ export default function PathMap() {
 
 // helpers //////////////////////
 
-// copied from https://github.com/adobe-webplatform/Snap.svg/blob/master/src/svg.js
-var tokenRegex = /\{([^}]+)\}/g,
+// copied and adjusted from https://github.com/adobe-webplatform/Snap.svg/blob/master/src/svg.js
+var tokenRegex = /\{([^{}}]+)\}/g,
```
Comment (philippfromme) [inline packages/dmn-js-drd/src/draw/PathMap.js:177]:
```suggestion
var tokenRegex = /\{([^{}]+)\}/g,
```

**Review by philippfromme — CHANGES_REQUESTED** [review]
Nice try!

File: packages/dmn-js-drd/src/draw/PathMap.js (hunk 1 above)
Comment (barmac) [reply packages/dmn-js-drd/src/draw/PathMap.js:177]: LOL

**Review by philippfromme — APPROVED** [review]
Good stuff! 👍🏻

**Pull request merged**